docs/src/main/sphinx/functions/conditional.rst

```diff
@@ -56,15 +56,9 @@ returned if it exists, otherwise null is returned. Example::
 IF
 --
 
-The ``IF`` function is actually a language construct
-that is equivalent to the following ``CASE`` expression:
-
-.. code-block:: text
-
-    CASE
-        WHEN condition THEN true_value
-        [ ELSE false_value ]
-    END
+The standard SQL ``IF`` expression has two forms, one supplying only a
```

Comment:
I think this is technically wrong so we probably should reword

Reply:
If this is incorrect then the opening line for the CASE section is also incorrect as they are more commonly referred to as statements (with expressions in them to be evaluated). Looks like David wrote the original, so I'd like to get his take then either fix both or leave both.